src/_locales/ru/messages.json

```diff
@@ -60,76 +60,76 @@
-    "html_options_suspend_hour": { "message": "час" },
+    "html_options_suspend_minute": { "message": "минуты" },
+    "html_options_suspend_minutes": { "message": "минут" },
+    "html_options_suspend_hour": { "message": "часа" },
     "html_options_suspend_hours": { "message": "часов" },
-    "html_options_suspend_day": { "message": "день" },
+    "html_options_suspend_day": { "message": "дня" },
     "html_options_suspend_days": { "message": "дней" },
-    "html_options_suspend_week": { "message": "неделю" },
-    "html_options_suspend_weeks": { "message": "недели" },
-    "html_options_suspend_no_pinned": { "message": "Никогда не приостанавливать закрепленные" },
+    "html_options_suspend_week": { "message": "недели" },
+    "html_options_suspend_weeks": { "message": "недель" },
+    "html_options_suspend_no_pinned": { "message": "Никогда не приостанавливать закрепленные вкладки" },
     "html_options_suspend_no_forms": { "message": "Никогда не приостанавливать вкладки с несохраненными полями ввода" },
     "html_options_suspend_no_audio": { "message": "Никогда не приостанавливать вкладки с проигрывающимся аудио" },
     "html_options_suspend_only_connected": { "message": "Никогда не приостанавливать вкладки в режиме оффлайн" },
     "html_options_suspend_only_on_battery": { "message": "Никогда не приостанавливать вкладки, когда подключен источник питания" },
-    "html_options_suspend_no_active_tabs": { "message": "Никогда не приостанавливать вкладки активную вкладку окна" },
-    "html_options_suspend_automatically_unsuspend": { "message": "Автоматически возобновлять вкладку при открытии" },
-    "html_options_suspend_discard_after_suspend": { "message": "Применить встроенный в Chrome механизм сохранения памяти при приостановке" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1a": { "message": "Уменьшает использование памяти приостановленной вкладки еще больше, используя" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1b": { "message": "встроенную в Chrome функциональность 'отбрасывание вкладки'. Это может привести к 500%" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1c": { "message": "уменьшению использования памяти при работе с большим количеством вкладок." },
-    "html_options_suspend_discard_after_suspend_tooltip_line2a": { "message": "Однако, это может также привести к небольшой задержке отображения при выборе" },
-    "html_options_suspend_discard_after_suspend_tooltip_line2b": { "message": "приостановленной вкладки." },
-    "html_options_suspend_discard_after_suspend_tooltip_line3a": {"message": "Для получения дополнительной информации об отбрасывании вкладок Chrome см.:"},
+    "html_options_suspend_no_active_tabs": { "message": "Никогда не приостанавливать активную вкладку в каждом окне" },
+    "html_options_suspend_automatically_unsuspend": { "message": "Автоматически возобновлять приостановленную вкладку при переходе на нее" },
```

Comment:
```suggestion
    "html_options_suspend_automatically_unsuspend": { "message": "Автоматически возобновлять приостановленную вкладку при переходе на неё" },
```

Reply:
here I agree

Reply:
Possibly missed "ё" here, "неё" instead of "нее", so as we decided to better use it.

@deanoemcke, could u please commit this to master so there's no need to create new PR just for one letter change.